spec/dummy_controller_spec.rb

```diff
@@ -1,3 +1,5 @@
+require 'support/shared_context/default_pagination_params'
+require 'support/shared_examples/proper_pagination_params'
```

Comment:
We could add this line to avoid requiring examples on specs:
```ruby
# spec_helper.rb
Dir[File.join('.', 'spec', 'support', '**', '*.rb')].each{ |f| require(f) }
```

Reply:
Yes, I knew it and I had it like that for a while, but then I thought: Why would I require all the support files in all of the specs, if just three of them need the shared examples/contexts I created? I know we do this in our projects, but I think would be also good to not require all those files in all the specs just to "not require them manually", because all the specs are loading things that not all of them need... What do you think about it?